frontend/src/components/buttons/Buttons.tsx

```diff
@@ -10,109 +9,60 @@ import { useGLTF } from "@react-three/drei";
 
 useGLTF.preload("/prev_button.glb");
 
-interface ButtonsProps {
-  playing: boolean,
-  setPlaying: () => void,
-  soundOn: boolean,
-  setLol: () => void
-}
-
-interface UnclickType {
-  [key: string]: (first: boolean) => void | boolean
-}
-
-export default function Buttons({ playing, setPlaying, soundOn, setLol } : ButtonsProps) {
```

Comment:
Got rid of these props mainly to prepare for the zustand functionality. Hoping the zustand change will be merged in first.